keypoints/rse-cmdline-git.md

```diff
@@ -0,0 +1,33 @@
+- Use git config with the `--global` option to configure a user name,
```

Comment:
More generic alternative: "Git can be configured on your computer to save your name, email, and other preferences."